ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/localzone/provision/cucushift-installer-rehearse-aws-ipi-ovn-localzone-provision-chain.yaml

```diff
@@ -10,6 +10,8 @@ chain:
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-aws-custom-vpc
   - ref: ipi-conf-aws
+  - ref: ovn-conf
```

Comment:
Per the logic of `ipi-conf-network-mtu`, sounds like we do not need the step.

Reply:
Or does that mean only OVN need to set custom MTU?

Reply:
Per our testing result, OVN will need NETWORK_MTU:1100, SDN may need a different MTU value. Considering we're testing the localzone in 4.12+ and it's using OVN as default network plugin, I think we can only test  `aws-ipi-localzone-provision` with OVN

Reply:
Got it. Per the new step - `ipi-conf-network-mtu` naming, I guess we can reuse it for other cases in the future. So my suggestion is:
1. removing `ovn-conf` step in this workflow, because we start localzone testing since 4.12, its default networkType is ovn.
2. rename the chain name to `cucushift-installer-rehearse-aws-ipi-network-mtu-localzone-provision`

Or if we want to explicitly show networking type in the step, maybe we can rename the `ipi-conf-network-mtu` to `ipi-conf-ovn-mtu`, and update the chain name accordingly.

Reply:
Right, the reason is the MTU value is different for OVN and SDN, the ovn/mtu-1100 is the pair in this workflow.

> Considering we're testing the localzone in 4.12+ and it's using OVN as default network plugin, I think we can only test `aws-ipi-localzone-provision` with OVN

Yes, AWS Local Zone is supported by 4.12+.

Reply:
>Or does that mean only OVN need to set custom MTU?
Actually this issue is not related openshift-sdn or OVN,  it's about the host from AZ to local zone only support MTU is 1300 in our testing 

```
sh-4.4# ping -M do -s 1300 -c 5 10.0.54.121
PING 10.0.54.121 (10.0.54.121) 1300(1328) bytes of data.
ping: local error: Message too long, mtu=1300
ping: local error: Message too long, mtu=1300
ping: local error: Message too long, mtu=1300
ping: local error: Message too long, mtu=1300
ping: local error: Message too long, mtu=1300

sh-4.4# ping -M do -s 1200 -c 5 10.0.54.121
PING 10.0.54.121 (10.0.54.121) 1200(1228) bytes of data.
1208 bytes from 10.0.54.121: icmp_seq=1 ttl=64 time=27.0 ms
1208 bytes from 10.0.54.121: icmp_seq=2 ttl=64 time=26.3 ms
1208 bytes from 10.0.54.121: icmp_seq=3 ttl=64 time=26.4 ms
1208 bytes from 10.0.54.121: icmp_seq=4 ttl=64 time=26.4 ms
1208 bytes from 10.0.54.121: icmp_seq=5 ttl=64 time=26.3 ms
```
So here whatever openshift-sdn or ovn,  MTU can be set 1100 I guess. 
even though 1150 also ok for openshift-sdn

Reply:
>Or if we want to explicitly show networking type in the step, maybe we can rename the ipi-conf-network-mtu to ipi-conf-ovn-mtu, and removing the sdn networking type judgement and codes, and update the chain name accordingly.

My suggestion is:  How about keeping the `sdn networking type judgement` here. if want to test sdn plugin with Local zone in future. So we just add another chain with step `ref: sdn-conf`

Reply:
I am okay with keep the snd netowrking type judgement, if that, does it make sense?
```
1. removing ovn-conf step in this workflow, because we start localzone testing since 4.12, its default networkType is ovn.
2. rename the chain name to cucushift-installer-rehearse-aws-ipi-network-mtu-localzone-provision
```

Reply:
> even though 1150 also ok for openshift-sdn

Per documentation's perspective, we need to provide a maximum MTU value (the suer setting must not be larger than this), so MTU values for SDN and OVN are different, and CI needs to match the value in the doc.

> 1. removing ovn-conf step in this workflow, because we start localzone testing since 4.12, its default networkType is ovn.
2. rename the chain name to cucushift-installer-rehearse-aws-ipi-network-mtu-localzone-provision

If we don't test SDN+LocalZone, I agree with this solution.

@jianlinliu @zhaozhanqi WDTY ^^ ?

Reply:
The issue was found when we running local zone testing, we have to adjust MTU, so personally I prefer to focus on localzone testing, but not focus on sdn/ovn + localzone. So I suggested  `cucushift-installer-rehearse-aws-ipi-network-mtu-localzone-provision`

Reply:
No objections,  @zhaozhanqi is it good for you?

Reply:
yes, it's ok for me